plugins/Diagnostics/Diagnostic/ServerInformational.php

```diff
@@ -28,8 +28,22 @@ public function execute()
     {
         $results = [];
 
-        if ( ! empty( $_SERVER['SERVER_SOFTWARE'] ) ) {
-            $results[] = DiagnosticResult::informationalResult('Server Info', $_SERVER['SERVER_SOFTWARE']);
+        if (!empty($_SERVER['SERVER_SOFTWARE'])) {
+
+            $rpd = new RequiredPrivateDirectories($this->translator);
+            $isGlobalConfigIniAccessible = $rpd->isGlobalConfigIniAccessible();
```

Comment:
You can't perform this check here. This information is also shown during the install. But at that state neither a `config.ini.php` is available, nor a piwik url is already set. Currently when trying to install Matomo the system check throws an exception as no database is configured.

Reply:
I've changed this so that the file access check will only be performed if matomo is already installed, so on the installation page the warnings will always be shown, but on the post-installation system check they will only be shown if config files are accessible. I've changed `isGlobalConfigIniAccessible()` to actually check access for `global.ini.php` not `config.ini.php` as they're both covered by the same rule.